README.md

```diff
@@ -111,6 +111,34 @@ xattr -d com.apple.quarantine <path-to-marksman-bin>
 brew install marksman
 ```
 
+### NixOS
```

Comment:
This should arguably be either:
1. a bullet point as 'Homebrew' above, or
2. a level 4 heading (and then 'Homebrew' should also be a level 4 heading).

Reply:
Yeah, you're right. I think it fits best next to the Homebrew install. Will change it 👍🏼